src/components/forms/hooks/useControlledInput.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+
+export default (
+  ref: React.MutableRefObject<HTMLInputElement>,
```

Comment:
Never used, not needed

Reply:
It's used in SearchBar in 
```
const { inputValue: query, onChangeInput, resetValue } = useControlledInput(
    searchInputRef,
    defaultValue,
    onChangeQuery,
  );
```

Reply:
Yes but the `ref` arg isn't used within `useControlledInput`